config/routes.rb

```diff
@@ -21,6 +21,7 @@
   post 'i-forgor/commit/:password_reset_token', to: 'users#commit_apply_new_password', as: 'forgor_apply_commit'
   get 'logout', to: 'session#destroy', as: 'logout'
   get 'browse', to: 'links#browse'
+  get 'links/:id/history', to: 'links#history'
```

Comment:
After a lot of googling I'm not actually 100% sure why actions such as "links/:id/edit" don't have an entry here--I assume something implicit is going on, I just haven't been able to identify _what_, Let me know what I'm missing about this.

Reply:
ok, i figured out that I was missing the "resources" call just a bit below this. don't ask why i didn't figure it out sooner